docs/prod-host-setup.md

```diff
@@ -167,3 +167,35 @@ Verify that swap is disabled by running:
 ```bash
 grep -q "/dev" /proc/swaps && echo "swap partitions present (Recommendation: no swap)" || echo "no swap partitions (OK)"
 ```
+
```

Comment:
I would make a section for Known kernel issues
Recommend to keep host kernel up to date.

Put there a recommendation for this CVE with following format:
Issue Description
 mention we would close this issue once solved in stable kernels
Impact
Vulnerable systems - checklist on how to find if a system is vulnerable
Mitigation
 if vulnerable update kernel once patch is provided
Recommendations if not vulnerable - there say to not update kernels...

Reply:
Done.